packages/nocodb/tests/unit/rest/tests/meta-apis/comment.test.ts

```
@@ -0,0 +1,174 @@
+    before(async function () {
+      context = await init();
+
+      base = await createProject(context);
+      table = await createTable(context, base);
+
+      const row = await createRow(context, { base, table });
+      firstRowId = row.Id;
+      expect(firstRowId).to.not.be.undefined;
+    });
+
+    it.only(`Comments Meta CRUD ${API_VERSION}`, async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)